src/platform-implementation-js/platform-implementation.ts

```diff
@@ -230,19 +229,6 @@ https://www.inboxsdk.com/docs/#RequiredSetup
     logger.eventSdkPassive('instantiate');
     const pi = new PlatformImplementation(driver, appId, opts);
 
-    // TODO: drop. This condition isn't possible.
-    if (origin === ('https://inbox.google.com' as any) && !opts.inboxBeta) {
```

Comment:
#887 removed most other references to old `'https://inbox.google.com'`. Without the `any`, TS correctly notes that this block can never be hit because of a check further up in the function.